plugin/src/main/groovy/com/btkelly/gnag/reporters/BaseViolationDetector.groovy

```diff
@@ -27,58 +25,4 @@ abstract class BaseViolationDetector implements ViolationDetector {
         this.project = project
     }
 
-    /**
-     * @param absoluteFilePath the absolute path to a target file. Should be trimmed of all excess whitespace and
-     *                         newlines. Can be null.
-     * @return the path to the target file, relative to the root directory of the current project, iff the target file
-     *         is contained within that directory; null otherwise
-     */
-    protected String computeFilePathRelativeToProjectRoot(final String absoluteFilePath) {
-        final String sanitizedAbsoluteFilePath = sanitizeToNonNull(absoluteFilePath)
-
-        final String expectedFilePathPrefix = project.getRootDir().getAbsolutePath() + "/"
-
-        if (!sanitizedAbsoluteFilePath.startsWith(expectedFilePathPrefix)) {
-            // The target file _is not_ contained within the root directory of the current project. Return null.
-            return null
-        } else {
-            /*
-             * The target file _is_ contained within the root directory of the current project. Return the path to the
-             * target file relative to the root directory.
-             */
-            return sanitizedAbsoluteFilePath.replace(expectedFilePathPrefix, "")
-        }
-    }
-
-    /**
-     * @param lineNumberString a String we would like to parse into an integer value.
-     * @param violationType the name of the violation currently being parsed. Used for logging only.
-     * @return a positive integer, if lineNumberString can be parsed into such a value; null in any other case.
-     */
-    protected Integer computeLineNumberFromString(final String lineNumberString, final String violationType) {
```

Comment:
Moved to allow Checkstyle parsing code (now shared by Checkstyle and ktlint checkers) to call these methods from outside the `reporters` package.